tests/integration.spec.coffee

```diff
@@ -1740,6 +1740,14 @@ describe 'SDK Integration Tests', ->
 						promise = resin.models.os.getLastModified('raspberrypi')
 						m.chai.expect(promise).to.eventually.be.an.instanceof(Date)
 
+					it 'should be able to query for a specific version', ->
+						promise = resin.models.os.getLastModified('raspberrypi', '1.24.0')
+						m.chai.expect(promise).to.eventually.be.an.instanceof(Date)
+
+					it 'should be able to query for a version containing a plus', ->
+						promise = resin.models.os.getLastModified('raspberrypi', '2.0.0+rev1')
```

Comment:
True, but we don't have any integration testing up here that isn't against the real service and releases, so I'm not sure how else to test this. Any ideas? Doesn't feel like a huge disaster if these start failing in future when we remove the images anyway - it'll be easy and clear how to fix them, and we'll have more time to work with.